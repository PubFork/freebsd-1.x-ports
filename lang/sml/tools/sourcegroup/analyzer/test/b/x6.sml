structure A = struct
  val x = B.y
end
