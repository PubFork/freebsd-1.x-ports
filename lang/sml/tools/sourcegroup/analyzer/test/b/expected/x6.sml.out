source sml "b/x6.sml"
  import structure B 
  export structure A ;

